dev-tools/releng/src/main.rs

```diff
@@ -307,24 +317,17 @@ async fn main() -> Result<()> {
                 Command::new(&args.git_bin)
                     .arg("-C")
                     .arg(&args.helios_dir)
-                    .args([
-                        "fetch",
-                        "--no-write-fetch-head",
-                        "origin",
-                        "master",
-                    ])
+                    .args(["fetch", "--no-write-fetch-head", "origin", "HEAD"])
```

Comment:
What does this change do?

Reply:
`HEAD` always refers to the default branch, if we ever end up renaming the branch. This is just a bit of unrelated futureproofing.

Reply:
I feel like it fetches whatever **HEAD** points to on the **origin** remote, without updating the local **FETCH_HEAD** ref?  I think **HEAD**, at least on GitHub, points at the tip of the default branch in the repository?

Reply:
Ah gotcha -- could you add a comment?